README.md

```diff
@@ -14,14 +14,14 @@ The calculator follows the semnatics and syntax of javascript and interactivity.
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://deploy-preview-1--made-by-js-calculator.netlify.app/)
```

Comment:
This should be your main deployment not your "deploy preview", thst is, https://made-by-js-calculator.netlify.app/, this url will be the one used on your master branch